packages/react-sdk/src/css/normalize.ts

```diff
@@ -124,7 +124,7 @@ export const body = [
     property: "fontFamily",
     value: {
       type: "keyword",
-      value: `system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'`,
+      value: "Arial, sans-serif",
```

Comment:
https://github.com/sindresorhus/modern-normalize/issues/3

We should use system-ui as default and expand into that list to mame ux consistent

Reply:
yeah, its a no fly, even they didn't merge